frontend/src/bundles/employers/components/sorting-dropdown/sorting-dropdown.tsx

```diff
@@ -0,0 +1,42 @@
+import { InputAdornment } from '@mui/material';
+import {
+    type Control,
+    type FieldPath,
+    type UseFormReset,
+} from 'react-hook-form';
```

Comment:
common bundle

Reply:
this is the common way to import rhf types in our project. Every component that needs these types (not just the common components) imports them directly. We currently do not have an export bundle for these types. Should we create it and change all the components?

Reply:
sorry it is types!